code/game/objects/effects/effect_system/effects_foam.dm

```diff
@@ -12,7 +12,7 @@
 	var/amount = 3
 	animate_movement = 0
 	var/metal = 0
-	var/lifetime = 6
+	var/lifetime = 40
```

Comment:
This renders the foam's reagent reactions basically useless. In order to avoid duplicating reagents, the amount that reacts during each processing cycle is the total volume divided by the total lifetime, so that the sum of all the reactions that happen is equal to the volume put in the foam.

With a lifetime of 40, it means that instead of having 6 reaction of Volume/6 , we now have 40 reaction of Volume/40. 
The issue is that the foam reagent reactions use the VAPOR method which check for clothes protecting the mob's skin. reaction_mob() is currently using a treshold (https://github.com/Iamgoofball/-tg-station/blob/68e752b46153e32d116d7a0d916de5c7c1ca199c/code/modules/reagents/chemistry/reagents.dm#L40) because even bio suits don't give a 100% protection so we need the treshold to not have small reagent amounts still reacting with the mob wearing a bio suit.
Now that you divide the reaction amount by 40 instead of 6, it means that any foam reaction with less than 20u of a given reagent won't have any effect, even on naked humans (and you probably need 40+ unit for people with just jumpsuits and shoes)